packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/ImagePickerModule.kt

```diff
@@ -266,10 +271,16 @@ class ImagePickerModule(private val mContext: Context,
       return
     }
 
-    val metadataRetriever = MediaMetadataRetriever().apply {
-      setDataSource(mContext, uri)
+    try {
+      val metadataRetriever = MediaMetadataRetriever().apply {
+        setDataSource(mContext, uri)
+      }
+      VideoResultTask(promise, uri, contentResolver, CacheFileProvider(mContext.cacheDir, ".mp4"), metadataRetriever).execute()
```

Comment:
@lukmccall should this always be `.mp4`? 🤔

Reply:
Yes, cause we currently don't support other formats on Android.